script.js

```diff
@@ -1,4 +1,122 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  // setting variables to avoid typo
+  var PAPER_OPTION = "paper";
+  var SCISSOR_OPTION = "scissor";
+  var STONE_OPTION = "stone";
+
+  // seetting variables for reverse SPS Game
+
+  var REV_STONE = "reversed stone";
+  var REV_PAPER = "reversed paper";
+  var REV_SCISSOR = "reversed scissor";
+
+  var generateComputerOption = function () {
+    var RandomNo = Math.random() * 3;
+    var randomSelection = Math.ceil(RandomNo);
+
+    if (randomSelection == 1) {
+      return SCISSOR_OPTION;
+    }
+    if (randomSelection == 2) {
+      return PAPER_OPTION;
+    }
+
+    return STONE_OPTION;
+  };
+
+  var computerOption = generateComputerOption();
+
+  // reversed SPS game winning conditions
+  // paper beats scissor
+  // stone beats paper
+  // scissor beats stone
+
+  if (
+    (input == REV_SCISSOR && computerOption == STONE_OPTION) ||
+    (input == REV_PAPER && computerOption == SCISSOR_OPTION) ||
+    (input == REV_STONE && computerOption == PAPER_OPTION)
+  ) {
+    return (
+      " The computer chose '" +
+      computerOption +
+      "' " +
+      "<br>" +
+      "You chose '" +
+      input +
+      "' " +
+      "<br>" +
+      " you won the reverse SPS game !"
+    );
```

Comment:
Consider having a global variable for the output text , so that with each `if `condition you can change the value of that global variable for output text, and you would just return that variable instead of a string of text, just like what you did below with `myOutputValue`